Pages/HomeScreen.xaml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="QuizzingApp341.Pages.HomeScreen"
```

Comment:
We'll probably want to standardize if screens end with 'Screen' or nothing eventually